PxWeb/Controllers/Api2/TableApiController.cs

```diff
@@ -177,11 +177,15 @@ private IActionResult GetData(string id, string? lang, VariablesSelection? varia
             }
             
             builder.BuildForPresentation(selection);
-            
+
             if (outputFormat == null)
             {
                 outputFormat = _configOptions.DefaultOutputFormat;
             }
+            else if (!_configOptions.OutputFormats.Contains(outputFormat))
```

Comment:
This compasison is case sansitive. It should be insensitive.

Reply:
Added StringComparer.OrdinalIgnoreCase